examples/platforms/cc2650/radio.c

```diff
@@ -178,78 +178,71 @@ static void rfCoreInitBufs(void)
  */
 static void rfCoreInitReceiveParams(void)
 {
-    memset((void *)&sReceiveCmd, 0x00, sizeof(sReceiveCmd));
-
-    sReceiveCmd.commandNo = CMD_IEEE_RX;
-    sReceiveCmd.status = IDLE;
-    sReceiveCmd.pNextOp = NULL;
-    sReceiveCmd.startTime = 0x00000000;
-    sReceiveCmd.startTrigger.triggerType = TRIG_NOW;
-    sReceiveCmd.condition.rule = COND_NEVER;
-    sReceiveCmd.channel = kPhyMinChannel;
-
-    sReceiveCmd.rxConfig.bAutoFlushCrc = 1;
-    sReceiveCmd.rxConfig.bAutoFlushIgn = 0;
-    sReceiveCmd.rxConfig.bIncludePhyHdr = 0;
-    sReceiveCmd.rxConfig.bIncludeCrc = 0;
-    sReceiveCmd.rxConfig.bAppendRssi = 1;
-    sReceiveCmd.rxConfig.bAppendCorrCrc = 1;
-    sReceiveCmd.rxConfig.bAppendSrcInd = 0;
-    sReceiveCmd.rxConfig.bAppendTimestamp = 0;
-
-    sReceiveCmd.pRxQ = &sRxDataQueue;
+    static const rfc_CMD_IEEE_RX_t cReceiveCmd =
+    {
+        .commandNo                  = CMD_IEEE_RX,
```

Comment:
I'm not sure how I feel about this notation (I also don't know if I have ever seen it used before), @jwhui what's your preference here?